Application/EdFi.Ods.Standard/Artifacts/Metadata/DatabaseViews.generated.json

```diff
@@ -923,504 +923,6 @@
       }
     ]
   },
-  {
```

Comment:
was this generated or manually edited?

Reply:
I definitely did not modify the `.generated.json` file manually. I went through the whole CI packaging and initdev process dance to achieve these results.